common/pipeline_storage/src/main/scala/uk/ac/wellcome/pipeline_storage/ElasticRetriever.scala

```diff
@@ -20,7 +20,8 @@ class ElasticRetriever[T](client: ElasticClient, index: Index)(
     extends Retriever[T]
     with Logging {
 
-  final def lookupSingleId(id: String): Future[T] = {
+  final def apply(id: String): Future[T] = {
+    debug(s"Looking up ID $id in index $index")
```

Comment:
`debug` messages don't get logged to ES and just make the tests more noisy so maybe worth removing?